source/module_esolver/esolver_ks_lcao.cpp

```diff
@@ -625,21 +625,23 @@ void ESolver_KS_LCAO::updatepot(const int istep, const int iter)
             }
             bool bit = false; // LiuXh, 2017-03-21
             // if set bit = true, there would be error in soc-multi-core calculation, noted by zhengdy-soc
-            if (this->psi != nullptr)
+            if (this->psi != nullptr && (istep % GlobalV::out_interval == 0))
```

Comment:
http://abacus.deepmodeling.com/en/latest/advanced/input_files/input-main.html#out-interval the parameter "out_interval" not support this print which described in documentation. 
@lyb9812 please discuss with @YuLiu98  for it.

Reply:
Ok, I will update the documentation.

Reply:
Parameter out_interval only used with "calculation == md" now.
What is your designation here?

Reply:
I added a function in esolver_ks_lcao.cpp in PR #2613, which might also be useful here.
```
bool ESolver_KS_LCAO::md_skip_out(std::string calculation, int istep, int interval)
 {
     if (calculation == "md")
     {
         if (istep % interval != 0)
         {
             return true;
         }
     }
     return false;
 }
```

Reply:
Yes,  out_interval is also only used with "calculation == md" for out_mat_hs and out_wfc_lcao.